packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts

```diff
@@ -18,17 +18,19 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { Color, StylePartial } from '../../types/lui';
+import { Color, StylePartial } from '../../types/lui';
 
-export type KnobStyle = {
+type KnobStyle = {
   circleColor: Color;
```

Comment:
is this right? circleColor only appears in tones/modes, so would it be accessible via `Knob.style.circleColor`?

Reply:
Yes I think circleColor property is accessible via Knob.style.circleColor, it is used in slider component.